integration_tests/pages/page.ts

```diff
@@ -15,14 +16,21 @@ export default abstract class Page {
       })
   }
 
-  constructor(private readonly title: string) {
+  constructor(private readonly title: string, private readonly backNavigationLink: BackLink = { hasBackLink: true }) {
     this.checkOnPage()
   }
 
   checkOnPage(): void {
     cy.get('h1').contains(this.title)
+    if (this.backNavigationLink.hasBackLink !== false) {
+      cy.get('[data-qa=back-link]').should('exist')
+    } else {
+      cy.get('[data-qa=back-link]').should('not.exist')
```

Comment:
meh.. could reuse `this.backNavigation()` in these couple of places

Reply:
thanks - sorted in latest commit